src/__tests__/showInApp.test.ts

```diff
@@ -1004,131 +1004,172 @@ describe('paginated', () => {
 });
 
 describe('setAsReadMode', () => {
-  describe('MANUAL', () => {
-    test('given unread count/badge, when popup in opened, unread count/badge do not clear', async () => {
-      notificationapi.showInApp({
-        root: 'root',
-        markAsReadMode: MarkAsReadModes.MANUAL
-      });
-      notificationapi.websocketHandlers.unreadCount({
-        route: 'inapp_web/unread_count',
-        payload: {
-          count: 5
-        }
+  [MarkAsReadModes.MANUAL, MarkAsReadModes.MANUAL_AND_CLICK].map((mode) => {
```

Comment:
No changes to tests here, just moved all the tests in this section into this array.